src/sgx/faasm_sgx_system.cpp

```diff
@@ -170,6 +201,10 @@ std::string faasmSgxErrorString(faasm_sgx_status_t status)
         ERROR_PRINT_CASE(FAASM_SGX_INVALID_PAYLOAD_LEN)
         ERROR_PRINT_CASE(FAASM_SGX_FUNCTION_NOT_WHITELISTED)
         ERROR_PRINT_CASE(FAASM_SGX_WAMR_FUNCTION_NOT_IMPLEMENTED)
+        // TODO - not do this manually
```

Comment:
For each error defined in `faasm_sgx_error.h`, we need to add an entry here so that the error text is printed. Adding a `TODO` so that we register these differently in the future.